### PR TITLE
add test_ink interface

### PR DESCRIPTION
**Pull request by shelvenzhou**

### CODE REVIEW

File: standalone/pruntime/enclave/src/lib.rs [hunk 1]
```diff
@@ -1527,6 +1538,513 @@ fn get_runtime_info(_input: &Map<String, Value>) -> Result<Value, Value> {
...
+            15, 58, 0, 0, 65, 0, 32, 1, 65, 1, 16, 3, 0, 11,
+        ],
+        initial_data: vec![248, 30, 126, 26, 0],
+        txs: vec![
+            vec![205, 228, 239, 169], // flip()
+            vec![109, 76, 230, 60],   // get()
+        ],
+    });
+    testcases.push(TestContract {
+        name: String::from("EIP20Token"),
```
Comment (h4x3rotab) [inline standalone/pruntime/enclave/src/lib.rs:1606]: ERC20?

File: standalone/pruntime/enclave/src/lib.rs [hunk 2]
```diff
@@ -1527,6 +1538,513 @@ fn get_runtime_info(_input: &Map<String, Value>) -> Result<Value, Value> {
...
+        code: vec![
```
Comment (h4x3rotab) [inline standalone/pruntime/enclave/src/lib.rs:1607]: Try [include_bytes](https://doc.rust-lang.org/std/macro.include_bytes.html)

File: standalone/pruntime/enclave/src/lib.rs (hunk 1 above)
Comment (shelvenzhou) [reply standalone/pruntime/enclave/src/lib.rs:1606]: yep

**Review by h4x3rotab — APPROVED** [review]
LGTM

**Pull request merged**